src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -657,16 +682,21 @@ public void addInfo(final AbstractLocusInfo<SamLocusIterator.RecordAndOffset> in
                 // the raw depth may exceed the coverageCap before the high-quality depth does. So stop counting once we reach the coverage cap.
                 if (unfilteredDepth < coverageCap) {
                     unfilteredBaseQHistogramArray[recs.getRecord().getBaseQualities()[recs.getOffset()]]++;
-                    unfilteredDepth++;
                 }
+                unfilteredDepth++;
```

Comment:
I think that this will lead to an OOB exception in line <HERE> below. Since the array is only allocated to coverageCap+1 in line 103. Please test with low MAX_COVERAGE.